BUILD.gn

```diff
@@ -22,6 +22,7 @@ declare_args() {
   skia_use_angle = false
   skia_use_egl = false
   skia_use_expat = true
+  esy_skia_enable_svg = false
```

Comment:
Thanks for prefixing this with `esy`, will make it easy to differentiate build steps we add 👍